core/chains/evm/txmgr/confirmer_test.go

```diff
@@ -3051,7 +3034,7 @@ func TestEthConfirmer_ResumePendingRuns(t *testing.T) {
 
 	pgtest.MustExec(t, db, `DELETE FROM pipeline_runs`)
 
-	t.Run("processes eth_txes with receipt older than minConfirmations that reverted", func(t *testing.T) {
+	t.Run("processes eth_txes with receipt before current head that reverted", func(t *testing.T) {
```

Comment:
If I understand what this was originally testing, I think `older than minConfirmations` should just mean finalized transactions. Inserting an older receipt might not do anything here now that the query checks for finalized txs.